style.css

```diff
@@ -109,3 +159,203 @@ header {
   right: 0;
   height: 250px;
 }
+
+.projects {
+  position: relative;
+}
+
+.projects-container {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  margin: 24px;
+}
+
+.projects-title {
+  flex-direction: column;
+  color: #212121;
+  max-width: 100%;
+}
+
+.pro-title,
+.intru-title {
+  font-family: Crete Round, sans-serif;
+  align-self: flex-start;
+  font-weight: 400;
+}
+
+.pro-title {
+  font-size: 3rem;
+  line-height: 40px;
+  width: 100%;
+}
+
+.underline {
+  background-color: #212121;
+  height: 1px;
+  width: calc(100% - 24px);
+}
+
+.multi-poste {
+  flex-direction: column;
+  margin-top: 80px;
+}
+
+.intru-title {
+  font-size: 32px;
+}
+
+.skills-language li {
+  min-width: 200px;
+}
+
+.multi-poste p,
+.languages > li {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.multi-poste ul li {
+  border: 1px solid #212121;
+}
+
+.btn-project {
+  width: 125px;
+}
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(1, 100% [col-start]);
+}
+
+.card-item {
+  position: relative;
+  flex-direction: column;
+  background-image: url(./image_icons/project_item.svg);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 550px;
+}
+
+.card {
+  padding: 36px;
+  padding-top: 32%;
+}
+
+.card > h3 {
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
```

Comment:
- [ ] Please keep your code DRY by avoiding redundancy of CSS attributes. In this case, `color: #fff` appears 8 times in your CSS. Please fix this by creating a particular class for this attribute. In the future you may use this tool to check for important CSS redundancies: https://courses.cs.washington.edu/courses/cse154/20su/resources/assets/css-redundancy-checker/index.html 
![image](https://user-images.githubusercontent.com/2739245/125145810-826abd80-e0e8-11eb-862f-d2c1b3f316a1.png)